client/src/ui/play/lobby.rs

```diff
@@ -26,5 +54,16 @@ pub fn draw_lobby(f: &mut Frame, area: Rect, lobby: &Lobby) {
         },
     );
     let players = List::new(encrypted_names).block(block);
-    f.render_widget(players, area);
+    f.render_widget(players, chunks[0]);
+
+    // Render a small help section for the lobby owner.
+    if lobby.local_player == lobby.owner && lobby.local_player.is_some() {
+        let commands = vec![format!(
+            "{} - Start the lobby",
+            app.config.key_bindings.lobby.start.to_string()
+        )];
+        let block = Block::bordered().title("Owner commands");
+        let command_list = List::new(commands).block(block);
+        f.render_widget(command_list, chunks[1]);
+    }
 }
```

Comment:
These should be dynamic. E.g., the start command should only be shown when the lobby is actually startable.